app/views/outfits/index.html.erb

```diff
@@ -1,10 +1,25 @@
 <h1>Your pet's next outfit...</h1>
 
+
+
 <div class="main-container d-flex align-top">
-  <div class="container d-flex justify-content-between mt-5">
-    <div class="card_grid_index">
-      <% @outfits.each do |outfit| %>
-        <%= render "/outfits/outfit_card", outfit: outfit%>
+  <div class="container mt-1">
+
+  <%= form_with url: outfits_path, method: :get, class: "d-flex" do %>
+  <%= text_field_tag :query,
```

Comment:
small design suggestion to fix in field input - it should be white and not the color of the body background so it stands out more!